### ISSUE TITLE
The code about RoBERTa model

### ISSUE THREAD

**Issue by YNT-Z** [wenyudu/MIGU#3]
Thank you for your work and look forward to the code on the RoBERTa model.

**Comment by wenyudu** [comment]
Hello! Thank you for your interest and for asking. The RoBERTa experiment isn't our primary experiment, so the code is somewhat disorganized right now. However, I've provided a detailed description below:

The implementation of our method for continual domain pre-training is primarily based on the DAS repository (https://github.com/UIC-Liu-Lab/ContinualLM). The dataset configuration, training sequence, and model training hyperparameters are consistent with those of DAS. 

Like the code we provided for T5, the main modification involves registering hook functions during the forward process to collect intermediate statistics of the dot product. Subsequently, after the model performs backpropagation and before parameter updates, we utilize these statistics to compute a gradient mask for the corresponding parameters, thereby constraining the gradients.
